src/main/java/it/gov/pagopa/payhub/activities/activity/ingestionflow/debtposition/SynchronizeIngestedDebtPositionActivityImpl.java

```diff
@@ -119,7 +108,60 @@ public String synchronizeIngestedDebtPosition(Long ingestionFlowFileId) {
         }
 
         log.info("Synchronization of all debt positions related to ingestion flow file id {} completed", ingestionFlowFileId);
-        return errors.toString();
+
+        String folderId = "";
+        if (!debtPositionsGenerateNotices.isEmpty()) {
+            folderId = generateMassive(ingestionFlowFileId, debtPositionsGenerateNotices);
+        }
+
+        return SyncIngestedDebtPositionDTO.builder()
+                .errorsDescription(errors.toString())
+                .folderId(folderId)
+                .build();
+    }
+
+    private List<Pair<DebtPositionDTO, WorkflowCreatedDTO>> getSyncDebtPositionAndWF(Long ingestionFlowFileId, PagedDebtPositions pagedDebtPositions, StringBuilder errors) {
```

Comment:
this is not a get operation
```suggestion
    private List<Pair<DebtPositionDTO, WorkflowCreatedDTO>> syncDebtPositions(Long ingestionFlowFileId, PagedDebtPositions pagedDebtPositions, StringBuilder errors) {
```